core/storage/suggestion/gae_models.py

```diff
@@ -485,6 +485,28 @@ def get_in_review_translation_suggestions(
             cls.language_code.IN(language_codes)
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions_by_exp_ids(
+        cls, exp_ids: List[str], language_code: str
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all in review translation suggestions matching the supplied
+        exp_ids and language_code.
+
+        Args:
+            exp_ids: list(str). Exploration IDs.
+            language_code: str. Language code.
```

Comment:
Please give more detail to explain the arg descriptions here and elsewhere. E.g. the language code here should be the language that the translation suggestions are in and not the original language of the exploration.